### PR TITLE
some minor command line cleanup of plfq and associated tools

### PR DESCRIPTION
**Pull request by timosachsenberg**

### CODE REVIEW

**Comment by jpfeuffer** [comment]
rebuild jenkins

File: src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp [hunk 1]
```diff
@@ -487,6 +486,7 @@ namespace OpenMS
     // store feature candidates before filtering
     if (!candidates_out_.empty())
     {
+      features.ensureUniqueId();
```
[the remark this answers is not in the document]
Comment (timosachsenberg) [reply src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp]: will move it out of the loop

File: src/utils/ProteomicsLFQ.cpp [hunk 2]
```diff
@@ -174,31 +176,64 @@ class UTILProteomicsLFQ :
     setValidStrings_("protein_quantification", ListUtils::create<String>("unique_peptides,strictly_unique_peptides,shared_peptides"));
 
 
-    registerStringOption_("targeted_only", "<option>", "false", "Only ID based quantification.", false, true);
+    registerStringOption_("targeted_only", "<option>", "false", "true: Only ID based quantification. false: include unidentified features.", false, false);
```
[the remark this answers is not in the document]
Comment (timosachsenberg) [reply src/utils/ProteomicsLFQ.cpp]: done

File: src/utils/ProteomicsLFQ.cpp [hunk 3]
```diff
@@ -174,31 +176,64 @@ class UTILProteomicsLFQ :
...
     setValidStrings_("targeted_only", ListUtils::create<String>("true,false"));
 
-    registerStringOption_("transfer_ids", "<option>", "false", "Requantification.", false, true);
-    setValidStrings_("transfer_ids", ListUtils::create<String>("false,merged,SVM"));
+    registerStringOption_("transfer_ids", "<option>", "false", "Requantification using mean RT if quantified by ID in more than 50% of all runs (of a fraction).", false, false);
```
[the remark this answers is not in the document]
Comment (timosachsenberg) [reply src/utils/ProteomicsLFQ.cpp]: done

**Pull request merged**